src/Engine/ProtoCore/Reflection/Mirror.cs

```diff
@@ -933,6 +933,22 @@ public List<string> GetArgumentNames()
                 return argNames;
             }
 
+            // List<ArgumentInfo> GetArguments() is used to get "Arguments"--> so that can use "Arguments" to get ArgumentsType;
+            //original method is  public List<ProtoCore.Type> GetArgumentTypes()
+            public List<ArgumentInfo> GetArguments()
```

Comment:
What if you just refer to the procNode.Arguments directly to whoever calls this?
Do you need to create a copy of the arguments?